cf_xarray/accessor.py

```diff
@@ -109,7 +109,7 @@
 
 #: regular expressions for guess_coord_axis
 regex = {
-    "time": "time[0-9]*",
+    "time": "time[0-9]*|min|hour|day|week|month|year",
```

Comment:
Can you add the attributes listed here: https://xarray.pydata.org/en/stable/generated/xarray.core.accessor_dt.DatetimeAccessor.html and https://xarray.pydata.org/en/stable/generated/xarray.core.accessor_dt.TimedeltaAccessor.html? (the ones that make sense anyway like `dayofyear`)